public/app/plugins/panel/stat/common.ts

```diff
@@ -94,8 +94,7 @@ export function addOrientationOption<T extends SingleStatBaseOptions>(
 ) {
   builder.addRadio({
     path: 'orientation',
-    name: 'Orientation',
-    description: 'Layout orientation',
+    name: 'Layout',
```

Comment:
This change impacts more than just the bar gauge panel (also stat, gauge, and maybe a few others)

I think we can leave this as is for time being given the increased surface area this change has 😬